### PR TITLE
Add the plan cmd to the test-cases tools to bulk update the target version

### PR DESCRIPTION
**Pull request by b1zzu**
# Description

Added two new cmds `./tools.sh plan release ...` and `./tools.sh plan for ...` that would help updating the target version on the test cases

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [x] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

**Comment by trepel** [comment]
It works as expected. I am not very familiar with typescript, but I looks good to me.

File: test-cases/README.md [hunk 1]
```diff
@@ -156,6 +156,24 @@ to pretty print the csv output on the terminal:
 ./tools.sh export csv | column -t -s, | less -S
 ```
 
+## Bulk update the target version on the test cases
+
+> Attention: all changes are applied locally and need to be manually pushed to the integreatly-operator repo
+
+To automatically add the target version to all test cases which target version is older then 3 releases use:
```
Comment (trdoyle81) [inline test-cases/README.md:163]:
```suggestion
To automatically add the target version to all test cases with a target version older than 3 releases, use the following command:
```

**Pull request merged**